src/api/config.ts

```diff
@@ -1,8 +1,8 @@
-export const baseUrl = 'https://admin.ylitse.fi/api';
-export const loginUrl = 'https://admin.ylitse.fi/login';
-export const termsUrl =
-  'https://sos-lapsikyla.fi/fileadmin/user_upload/tiedostot/kuvat/pdf-tiedostot/Ylitse_MentorApp_kayttyoehdot_tietosuojaseloste.pdf';
-export const userGuideUrl =
-  'https://www.sos-lapsikyla.fi/fileadmin/user_upload/tiedostot/kuvat/pdf-tiedostot/Ylitse-vertaismentorointiopas-web.pdf';
-export const feedBackUrl =
-  'https://forms.office.com/Pages/ResponsePage.aspx?id=UGBhp7WfhU21Tu69_G6kO5HyhsO7ItpBu8SARGpVxBdURTBPRk05Wk8yMTlNTlZFRERNTjY4N0lPMC4u';
+import *  as config from '../../config.json'
+import *  as localConfig from '../../config.local.json'
+
+export const baseUrl = localConfig.baseUrl ? localConfig.baseUrl: config.baseUrl
+export const loginUrl = localConfig.loginUrl ? localConfig.loginUrl : config.loginUrl 
+export const termsUrl = localConfig.termsUrl ? localConfig.termsUrl : config.termsUrl
+export const userGuideUrl =  localConfig.userGuideUrl ? localConfig.userGuideUrl :config.userGuideUrl
+export const feedBackUrl =  localConfig.feedBackUrl ? localConfig.feedBackUrl : config.feedBackUrl
```

Comment:
would it be simpler to just have one config.ts and say for tell users to copy it? Flow to release would now be to put an empty strings to local config.

With this you anyway have to fill config.json too to get things to compile.

Reply:
I mean one config.json

Reply:
Yeah, since we'll already be copying local config as per README, only the main config file needs to be imported.